### PR TITLE
Creates ANS abstract template

### PR DESCRIPTION
**Pull request by samgdotson**
This PR includes all of the files required to make an abstract for ANS. Replete with terrible jokes for your amusement.

### CODE REVIEW

**Comment by samgdotson** [comment]
@nsryan2

**Comment by nsryan2** [comment]
@samgdotson  
Are there different types of ANS abstract? The Student conference abstract for a poster had a different format, and the TOFE 2020 Abstract does as well. Both appeared as a title with author and institution lines, and then paragraphs of text. If this is the case, is there a general purpose for this template that we could name it after to avoid confusion because I see what you were saying about how this template is on the ANS site as well?

**Comment by samgdotson** [comment]
Technology of Fusion Energy (TOFE) is not the general meeting, it's an "embedded topical."
The abstract for a poster is typically just a paragraph of information, it appears to be the same for TOFE. 
Sadly, there is no "general format" that would allow us to capture every variation in a single template repository. Fortunately, the outliers you mention are just paragraphs of text and don't need their own repositories. 

This template will be used for: The annual meeting, winter meeting, and student conference.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
 # ans-abs-template
-Template for American Nuclear Society conference abstracts
+
+Template for American Nuclear Society conference abstracts.
```
Comment (nsryan2) [inline README.md]:
```suggestion
Template for the American Nuclear Society Annual Meeting, Winter Meeting, and Student Conference abstracts.
```

**Review by nsryan2 — COMMENTED** [review]
Oh, I see the difference. I think this would be a good note to add to the readme file. Otherwise, it looks good to me.

**Comment by katyhuff** [comment]
.... adding labels....

**Comment by katyhuff** [comment]
...adding reference to relevant issue.... This is part of the work in https://github.com/arfc/arfc.github.io/issues/160

**Comment by katyhuff** [comment]
... adding project ....

File: README.md [hunk 2]
```diff
@@ -1,2 +1,15 @@
 # ans-abs-template
-Template for American Nuclear Society conference abstracts
+
+Template for the American Nuclear Society Annual Meeting, Winter Meeting, and Student Conference abstracts.
```
Comment (katyhuff) [inline README.md]: Please hard wrap lines at 80 chars
```suggestion
Template for the American Nuclear Society Annual Meeting, Winter Meeting, and 
Student Conference abstracts.
```

File: acknowledgements.tex [hunk 3]
```diff
@@ -0,0 +1,12 @@
+\section{Acknowledgments}
+
```
Comment (katyhuff) [inline acknowledgements.tex]:
```suggestion
Find the appropriate acknowledgements text at 
[the how-to-acknowledge Google Doc](https://docs.google.com/spreadsheets/d/1VybV0oMPiqpInTwgO0ICq0EC4AL_fGj-zbXGJAhRiVU/edit?usp=sharing)
```

**Review by katyhuff — CHANGES_REQUESTED** [review]
Thanks @samgdotson

**Comment by samgdotson** [comment]
@nsryan2 

nudge

**Review by nsryan2 — APPROVED** [review]
Looks good to me! What is the change it says is still requested? I see the changes to the readme and acknowledgements, but I'm not sure what the third one is.

File: acknowledgements.tex [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+[the how-to-acknowledge Google Doc](https://docs.google.com/spreadsheets/d/1VybV0oMPiqpInTwgO0ICq0EC4AL_fGj-zbXGJAhRiVU/edit?usp=sharing)
+This work was made possible by the wonderful people in ARFC. Additionally,
+this work is funded by [my funding source].
+Prof. Huff is supported by the Nuclear Regulatory Commission Faculty
+Development Program (award NRC-HQ-84-14-G-0054 Program B), the Blue Waters
+sustained-petascale computing project supported by the National Science
+Foundation (awards OCI-0725070 and ACI-1238993) and the state of Illinois,
+the DOE ARPA-E MEITNER Program (award DE-AR0000983), the DOE H2@Scale
+Program (award ), and the International Institute for Carbon Neutral Energy
+Research (WPI-I2CNER), sponsored by the Japanese Ministry of Education,
```
Comment (katyhuff) [inline acknowledgements.tex]: out of date. Please see the [the how-to-acknowledge Google Doc](https://docs.google.com/spreadsheets/d/1VybV0oMPiqpInTwgO0ICq0EC4AL_fGj-zbXGJAhRiVU/edit?usp=sharing)

File: main.tex [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+\usepackage{subcaption}
+\usepackage{enumitem}
+\usepackage{placeins}
+\usepackage{tikz}
+\usetikzlibrary{shapes.geometric, arrows}
+\usepackage[acronym,toc]{glossaries}
+\include{acros}
+\makeglossaries
+
+\newcolumntype{c}{>{\hsize=.56\hsize}X}
```
Comment (katyhuff) [inline main.tex]:
```suggestion
```

File: main.tex [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+\newcolumntype{b}{>{\hsize=.7\hsize}X}
```
Comment (katyhuff) [inline main.tex]:
```suggestion
```

File: main.tex [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+\newcolumntype{s}{>{\hsize=.74\hsize}X}
```
Comment (katyhuff) [inline main.tex]:
```suggestion
```

File: main.tex [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+\newcolumntype{f}{>{\hsize=.1\hsize}X}
```
Comment (katyhuff) [inline main.tex]:
```suggestion
```

File: main.tex [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+\newcolumntype{a}{>{\hsize=.45\hsize}X}
```
Comment (katyhuff) [inline main.tex]:
```suggestion
```

File: main.tex [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+\usepackage{titlesec}
+\titleformat*{\subsection}{\normalfont}
```
Comment (katyhuff) [inline main.tex]: This seems like maybe it undoes something in the ANS style?

File: main.tex [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+\usepackage[acronym,toc]{glossaries}
+\include{acros}
+\makeglossaries
+
+\newcolumntype{c}{>{\hsize=.56\hsize}X}
+\newcolumntype{b}{>{\hsize=.7\hsize}X}
+\newcolumntype{s}{>{\hsize=.74\hsize}X}
+\newcolumntype{f}{>{\hsize=.1\hsize}X}
+\newcolumntype{a}{>{\hsize=.45\hsize}X}
+\usepackage{titlesec}
```
Comment (katyhuff) [inline main.tex]: This seems like maybe it undoes something in the ANS style?

**Review by katyhuff — CHANGES_REQUESTED** [review]

File: main.tex (hunk 10 above)
Comment (samgdotson) [reply main.tex]: Yes, it unbolds subsection font.

File: main.tex (hunk 11 above)
Comment (samgdotson) [reply main.tex]: This doesn't make any visible changes to the document style.

File: acknowledgements.tex [hunk 12]
```diff
@@ -0,0 +1,13 @@
...
+
+Find the appropriate acknowledgements text at
+\href{https://docs.google.com/spreadsheets/d/1VybV0oMPiqpInTwgO0ICq0EC4AL_fGj-zbXGJAhRiVU/edit?usp=sharing}{the how-to-acknowledge Google Doc}
+
+Prof. Huff is supported by the Nuclear Regulatory Commission Faculty
+Development Program (award NRC-HQ-84-14-G-0054 Program B), the Blue Waters
+sustained-petascale computing project supported by the National Science
+Foundation (awards OCI-0725070 and ACI-1238993) and the state of Illinois, the
+DOE ARPA-E MEITNER Program (award DE-AR0000983), and the DOE H2@Scale Program
+(Award Number: DE-EE0008832)
```
Comment (nsryan2) [inline acknowledgements.tex:13]: This section is up-to-date, but the review from Prof. Huff makes me think that adding a note below this part saying something like, "The above section can be found in the how-to-acknowledge document, and you should check to ensure that this acknowledgment is accurate."

**Comment by katyhuff** [comment]
@samgdotson will handle outstanding review comments (update from group meeting)

**Review by katyhuff — APPROVED** [review]

**Comment by katyhuff** [comment]
Nice work!!

**Pull request merged**